# Generated by Django 3.1.3 on 2020-11-25 15:53

import django.contrib.auth.validators
from django.db import migrations, models
import django_extensions.db.fields


class Migration(migrations.Migration):

    dependencies = [
        ('core', '0003_auto_20201125_1551'),
    ]

    operations = [
        migrations.AddField(
            model_name='teammember',
            name='inner_username',
            field=django_extensions.db.fields.AutoSlugField(blank=True, editable=False, populate_from=('first_name', 'last_name'), unique=True, verbose_name='slug'),
        ),
        migrations.AlterField(
            model_name='teammember',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
